internal/services/automation/automation_account_resource.go

```diff
@@ -58,6 +60,50 @@ func resourceAutomationAccount() *pluginsdk.Resource {
 
 			"identity": commonschema.SystemAssignedUserAssignedIdentityOptional(),
 
+			"encryption": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*schema.Schema{
+						"user_identity_id": {
```

Comment:
Can we rename this to be consistent with other encryption blocks within the provider?
```suggestion
						"user_assigned_identity_id": {
```